dspace-xmlui/src/main/webapp/WEB-INF/web.xml

```diff
@@ -101,19 +101,22 @@
   <!--
     - Declare a filter to force UTF-8 encoding for all servlet requests
     -->
-  <filter>
-    <filter-name>SetCharacterEncoding</filter-name>
-    <filter-class>org.dspace.app.xmlui.cocoon.SetCharacterEncodingFilter</filter-class>
```

Comment:
forceEncoding (which the custom filter lacks) will unconditionally set the encoding of the response.  However it also does this to the request, which may have undesirable effects.

http://docs.spring.io/spring-framework/docs/2.5.x/api/org/springframework/web/filter/CharacterEncodingFilter.html